cmd/yurt-controller-manager/controller-manager.go

```diff
@@ -38,7 +38,8 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	newRand := rand.New(rand.NewSource(time.Now().UnixNano()))
+	newRand.Seed(time.Now().UnixNano())
```

Comment:
would you be able to explain the reason of change rand seed?

Reply:
see desc

Reply:
good job.